CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## master / unreleased
 
+* [CHANGE] Update the default HTTP configuration values for the S3 client to the upstream Thanos default values. #3244
```

Comment:
I would make it clear it affects the blocks storage.

```suggestion
* [CHANGE] Blocks storage: update the default HTTP configuration values for the S3 client to the upstream Thanos default values. #3244
```

Reply:
👍